Sources/MongoSwift/Operations/FindOperation.swift

```diff
@@ -32,6 +32,10 @@ public enum CursorType {
 
 /// Options to use when executing a `find` command on a `MongoCollection`.
 public struct FindOptions: Codable {
+    /// Enables writing to temporary files. When set to true, the find operation
```

Comment:
We may want to clarify that this option allows the _server_ to write to temporary files.

Reply:
I added the clarification to the aggregation option as well.
Also hopefully I finally got my commit hook right and that'll be the last lint fix commit I gotta make. 😅